inference-engine/tests/functional/plugin/myriad/subgraph_tests/nonzero_broadcast.cpp

```diff
@@ -19,6 +22,13 @@ class NonZero_Broadcast : public testing::WithParamInterface<BroadcastExplicitTe
                           public LayerTestsUtils::LayerTestsCommon {
 protected:
     void SetUp() override {
+        SetRefMode(LayerTestsUtils::RefMode::CONSTANT_FOLDING);
+        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
+        // DISABLE_REORDER is needed for Myriad2 cases
+        if (CommonTestUtils::vpu::CheckMyriad2()) {
```

Comment:
Comment is useless. It either should explain a reason of incompatibility with MY2 or it's better to remove it